src/js/index.js

```diff
@@ -46,28 +47,33 @@ function handleDifficulty(difficultyChanged) {
 	return false;
 }
 
-function handleTags() {
-	const tags = selectr.getValue();
+function updateUrl() {
```

Comment:
Since we already have a `handleTags` function, I think it might be better to add this in:

```js
for(let [key, value] of search) {
	if (!value.trim()) {
		search.delete(key);
	}
}
```

and drop the conditionals in the other parts of the file